### DISCUSSION TITLE
Deployments stuck in "Queued"

### DISCUSSION THREAD

**Discussion by sbmsr** [vercel/community#148] [Help] [answered]
Hi all,

My code has stopped deploying after each commit. I see the deployments are stuck in "Queued" for more than 30 minutes and counting. We have verified this issue across multiple projects as well. We do not have any concurrent builds. Any support is appreciated.

**Comment by sbmsr** [comment]
The issue resolved itself - there were some issues with builds according to [vercel-status.com](https://vercel-status.com)

**Reply by LatentM** [reply]
> The issue resolved itself - there were some issues with builds according to [vercel-status.com](https://vercel-status.com)
What does that mean? How to check it for my website?

**Comment by danielbixel** [comment]
I am facing a simular issue today.

2 hours ago a deployment worked and completed. Then suddenly the following deployments ended up in the queue.
I canceled the deployments and tried again. As far as I can see I have no running deployments now but each new try gets queued.

Anyone experiencing the same? - Any suggestions?

**Reply by k-tten** [reply]
+1 same

**Reply by LatentM** [reply]
I am also facing the same issue

**Reply by fu-sen** [reply]
I was experiencing this in a build a few days ago, but I haven't seen any comments here yet at that time. Perhaps the build operation is temporarily unstable for several days.

**Reply by LatentM** [reply]
> I was experiencing this in a build a few days ago, but I haven't seen any comments here yet at that time. Perhaps the build operation is temporarily unstable for several days.

So the only solution is to wait?

**Reply by fu-sen** [reply]
Yes. If we continue to use Vercel, we will have to wait.

**Reply by davidroberto** [reply]
Same here...

**Reply by mukulrajpoot262610** [reply]
same here

**Comment by xiaohk** [comment]
Same here today :(

**Comment by samirasaad** [comment]
the same today !

**Comment by fu-sen** [comment]
Vercel status has been updated. Vercel members will be aware of this situation.
https://www.vercel-status.com/

**Comment by micahlt** [comment]
Same issue here today.

**Comment by sufiyank97** [comment]
Same issue here today.

**Comment by KamilKubik** [comment]
Same issue here today, don't you worry! The issue is on the vercel side. Follow this site: https://www.vercel-status.com

**Comment by mukulrajpoot262610** [comment]
same issue

**Comment by NadsonKT** [comment]
Entrei em contato direito com a Vercel, o suporte retornou que o problema logo será resolvido.

**Comment by fungilation** [comment]
I have my build stuck on Queued too. Is it Vercel, or is it me sticking 6000 /public small files in my commit? 🤔

**Comment by leerob** [comment, marked as answer]
NPM is having issues https://www.vercel-status.com/

**Reply by juanpasolano** [reply]
Thanks! 👀

**Comment by fungilation** [comment]
NPM always have issues

**Comment by MAfzalKhan1997** [comment]
experiencing same today for last 30 minutes 😶

**Reply by dannyghub** [reply]
My build started just now (after 20min wait)

**Reply by dannyghub** [reply]
It's not looking good, it's now hung on `Installing build runtime...` for 4 minutes (usually takes 2.* seconds) - 😞

**Reply by MAfzalKhan1997** [reply]
yup brother... mine stuck too, no progress still.

**Reply by ManalLiaquat** [reply]
same here mate! 😥

**Reply by ManalLiaquat** [reply]
![image](https://user-images.githubusercontent.com/36041100/146221967-faa7b101-a502-4259-a456-1b8af38afa10.png)

status right now: https://www.vercel-status.com/

**Reply by genelorenzSarmiento0408** [reply]
Same here

**Reply by genelorenzSarmiento0408** [reply]
Oh It works now I guess 🤷‍♂️

**Reply by ManalLiaquat** [reply]
![image](https://user-images.githubusercontent.com/36041100/146226527-0f86aafa-5b53-499d-bdd9-a094ca2d09b3.png)

fixed!

**Comment by Pieparker** [comment]
Thank you all for reaching out. However, this topic is rather old and has been marked as Answered.

If you’re exhibiting the same or a similar issue to the original poster, please feel free to create your own discussion and provide as much context as you can. This helps us to ensure that discussions are relevant, fresh and free of any possibly misleading context.
